part2/stages/TPM-Timeout

```diff
@@ -0,0 +1,52 @@
+#!/bin/ash
+#
+# Copyright (c) 2013 Citrix Systems, Inc.
```

Comment:
TPM-Timeout is a modified version of another TPM file.  Will add additional Copyright statement.